CoreScriptsRoot/Modules/BackpackScript3D.lua

```diff
@@ -137,6 +162,7 @@ end
 
 local function OnCharacterAdded(character)
 	local backpack = player:WaitForChild("Backpack")
+	local humanoid = character:WaitForChild("Humanoid")
```

Comment:
Some characters in games never get a humanoid, which would make this callback yield forever.